src/components/panel/panel.spec.js

```diff
@@ -122,6 +122,36 @@ describe('$mdPanel', function() {
     expect(PANEL_WRAPPER_CLASS).not.toHaveClass(HIDDEN_CLASS);
   });
 
+
+  it('should destroy the panel when closing', function () {
+    openPanel(DEFAULT_CONFIG);
+
+    expect(panelRef._panelContainer).not.toHaveClass(HIDDEN_CLASS);
+    expect(panelRef.isAttached).toEqual(true);
+    expect(panelRef._config.locals).not.toEqual(null);
+
+    panelRef.close();
+
+    $rootScope.$apply();
+
+    expect(panelRef.isAttached).toEqual(false);
+    expect(panelRef._config.locals).toEqual(null);
+  });
+
+  it('should focus on origin element after hide (if provided)', function () {
+    attachToBody('<button id="donuts">Donuts</button>');
+
+    var detachFocusConfig = angular.extend({ origin: '#donuts' }, DEFAULT_CONFIG);
+
+    openPanel(detachFocusConfig);
+
+    panelRef.hide();
+
+    $rootScope.$apply();
+
+    expect(angular.element(document.activeElement).attr('id')).toBe('donuts');
```

Comment:
The `toBeFocused` matcher seems appropriate here. If it's not in the spec file, go ahead and add it.

You should also have an expect that focus is elsewhere before the hide event. With only this expect, I don't know if focus was never directed to the panel. (Likely it wasn't.)